src/client/components/app/Transactions/StakeTx.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import { t } from 'i18next';
```

Comment:
import the `useAppTranslation` hook instead, and use it as
```const { t } = useAppTranslation('common');```

Reply:
Refactored with https://github.com/yearn/yearn-finance-v3/pull/741/commits/95423b1a57b0eb31487b4169052c6c7b0e145478